charmhelpers/contrib/storage/linux/ceph.py

```diff
@@ -1185,6 +1185,17 @@ def __init__(self, api_version=1, request_id=None):
             self.request_id = str(uuid.uuid1())
         self.ops = []
 
+    def add_op(self, op):
+        """Add an op if it is not already in the list.
+
+        :param op: Operation to add.
+        :type op: dict
+        """
+        for existing_op in self.ops:
+            if op == existing_op:
+                return
+        self.ops.append(op)
```

Comment:
Won't this do the same:

if op not in self.ops:
   self.ops.append(op)

Reply:
Err, yes, thats much nicer.